java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -330,6 +330,7 @@ private void handleBlockExit(ProgramPoint programPosition) {
       pair.b.stream().forEach(s -> enqueue(truePP, s));
     } else {
       //  Table switch and lookup
+      // TODO : filter some node of the EG depending of the exceptionType in the successor.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)